R/teal_data-class.R

```diff
@@ -0,0 +1,103 @@
+setOldClass("JoinKeys")
+
+#' Reproducible data class.
+#'
+#' Reproducible data class basing on [`teal.code::qenv-class`].
+#' Don't interact with slots directly, it is suggested to use methods instead
+#' (see `methods(class = "teal_data")`).
```

Comment:
This feels a little aggressive. How about: please see methods to interact with class' slots.

Reply:
> I let you suggest the whole phrase.

Don't mind if I do :wink: